### ISSUE TITLE
Feature request: Implement actiona daemon to auto execute actions on dbus events

### ISSUE THREAD

**Issue by MurzNN** [Jmgr/actiona#174]
Will be good to integrate into Actiona a tool to automate launching scripts by system events (dbus on Linux and events on Windows), is it possible? If something like this already exists, please describe how to use it.
And if such tool is already exists as separate project, please point to it. Thanks!

**Comment by Jmgr** [comment]
Yes, this is something I've had in my wish list for a long time, some sort of execution management tool. This is unfortunately quite complicated to do, especially if you want to do it in cross-platform way. For now I am focusing on getting https://github.com/Jmgr/actiona/pull/143 ready.

You could use system-specific tools, for instance on Windows you can use [Windows Task Scheduler](https://en.wikipedia.org/wiki/Windows_Task_Scheduler) to run [actexec](https://wiki.actiona.tools/doku.php?id=en:actexec) with a script. I'm sure there must be something similar for Linux distributions.

**Comment by Jmgr** [comment]
Actiona 4 issue: https://github.com/Jmgr/actiona4/issues/24
